contrib/kind.sh

```diff
@@ -92,6 +93,7 @@ usage() {
     echo "-dd  | --dns-domain                 Configure a custom dnsDomain for k8s services, Defaults to 'cluster.local'"
     echo "-cn  | --cluster-name               Configure the kind cluster's name"
     echo "-ric | --run-in-container           Configure the script to be run from a docker container, allowing it to still communicate with the kind controlplane" 
+    echo "-ehp | --egress-ip-healthcheck-port TCP port used for gRPC session by egress IP node check. DEFAULT: 0 (no gRPC, just dial port 9)."
```

Comment:
you should default this to a port. That way we will get proper CI coverage with the e2e control plane tests. In the future maybe we should discuss defaulting this for ovnk altogether, but we probably need to discuss that with the rest of the community. For now it should be safe to enable it in kind.